unitTest/@testFEMComputationalFluidDynamicsAnalysis/testFEM4NavierStokesSteadyStateFlowAroundCylinder2D.m

```diff
@@ -0,0 +1,138 @@
+%
+% Function layout :
+%
+% 0. Read input
+%
+% 1. Parse the data from the GiD input file
+%
+% 2. GUI
+%
+% 3. Choose the equation system solver
+%
+% 4. Define the name of the vtk file from where to resume the simulation
+%
+% 5. Change the input velocity to match the reference paper - parabolic input
+%
+% 6. Solve the CFD problem
+%
+% 7. Calculate drag and lift force from the nodal forces
+%
+% 8. Calculate drag and lift coefficient based on drag and lift force
+%
+% 9. Define the expected solutions
+%
+% 10. Verify the results
+%
+%% Function main body
+
+%% 0. Read input
+% Define absolute tolerance
+absTol = 0.04; % tolerance w.r.t the reference paper
+absTol2 = 1e-10; % tolerance w.r.t our value
+
+% Define the path to the case
+pathToCase = '../../inputGiD/FEMComputationalFluidDynamicsAnalysis/';
+caseName = 'unitTest_testFEM4NavierStokesSteadyStateFlowAroundCylinder2D';
+
+%% 1. Parse the data from the GiD input file
+[fldMsh,homDBC,inhomDBC,valuesInhomDBC,nodesALE,~,analysis,parameters,...
+    propNLinearAnalysis,propFldDynamics,gaussInt,~] = ...
+    parse_FluidModelFromGid...
+    (pathToCase,caseName,'');
+
+%% 2. GUI
+% On the body forces
+computeBodyForces = @computeConstantVerticalBodyForceVct;
+
+% On the initial conditions
+% computeInitialConditions = @computeInitialConditionsFromVTKFileFEM4NSE2D;
+computeInitialConditions = @computeNullInitialConditionsFEM4NSE2D;
+
+% On the transient analysis properties
+if strcmp(propFldDynamics.method,'bossak')
+    propFldDynamics.computeProblemMtrcsTransient = ...
+        @computeProblemMtrcsBossakFEM4NSE;
+    propFldDynamics.computeUpdatedVct = ...
+        @computeBossakTIUpdatedVctAccelerationFieldFEM4NSE2D;
+end
+
+%% 3. Choose the equation system solver
+if strcmp(analysis.type,'NAVIER_STOKES_2D')
+    solve_LinearSystem = @solve_LinearSystemMatlabBackslashSolver;
+elseif strcmp(analysis.type,'NAVIER_STOKES_3D')
+    solve_LinearSystem = @solve_LinearSystemGMResWithIncompleteLUPreconditioning;
+else
+    error('Neither NAVIER_STOKES_2D or NAVIER_STOKES_3D has been chosen');
+end
+
+%% 4. Define the name of the vtk file from where to resume the simulation
+VTKResultFile = 'undefined';
+
+%% 5. Change the input velocity to match the reference paper - parabolic input
+
+% max input velocity defined in the reference paper
+Umax = 0.3;
+
+% change the input velocity to have the parabolic distribution
+valuesInhomDBCModified = inputInletVelocityParabolic(fldMsh, inhomDBC, valuesInhomDBC, Umax);
+
+%% 6. Solve the CFD problem
+[~,FComplete,hasConverged,~] = solve_FEMVMSStabSteadyStateNSE2D...
+    (fldMsh,homDBC,inhomDBC,valuesInhomDBCModified,nodesALE,parameters,...
+    computeBodyForces,analysis,computeInitialConditions,...
+    VTKResultFile,solve_LinearSystem,propFldDynamics,propNLinearAnalysis,...
+    gaussInt,caseName,'');
+
+%% 7. Calculate drag and lift force from the nodal forces
+[Fx, Fy, ~] = calculateForce(FComplete, fldMsh, homDBC, parameters, 2);
```

Comment:
This function is no more necessary: As the input file of this unit test case will be updated to account for the newly added variable postProc, the function which will be called will be parsed by the parser accordingly. I would call the function for this case computeLiftForce in case it should return the lift, computeDragForce in case it should return the drag or computeDragAndLiftForce in case it should return both. The function should then return a vector F which will have only one component in case only the lift or the drag is to be returned or n components if it is supposed to return n types of forces. The latter function computeBlaBlaBla should be always user defined and be saved in a new directory under FEMComputationalFluidDynamicsAnalysis with the name postProcessing. For the case of the unit test, I would add at the end of the function the postfix _unitTest so that it is clear that it is used for a unit tests and should not be modified, e.g. computeBlaBlaBla_unitTest.